AutorunsToWinEventLog/AutorunsToWinEventLog.ps1

```diff
@@ -1,8 +1,15 @@
-# Author: Chris Long (@Centurion)
+# Authors: Chris Long (@Centurion), Eric (@vector_sec)
 # This script executes the Sysinternals Autoruns CLI utility and saves the output to a CSV.
-# The resulting CSV entries are written to a Windows Event Log called "Autoruns"
+# Additionally, this script enumerates desktop shortcuts, BITS jobs, and browser extensions.
+# The discovered entries from all tools are written to a Windows Event Log called "Autoruns"
+
 
 ## Code to create the custom Autoruns Windows event log if it doesn't exist
+# The following event IDs are in use:
+# 1 - Sysinternals Autoruns results
+# 2 - BITS jobs
```

Comment:
Numbering will need to be updated as this PR reserved #2: https://github.com/palantir/windows-event-forwarding/pull/30